packages/libs/components/src/map/MapVEuMap.tsx

```diff
@@ -441,7 +441,7 @@ function CustomZoomControl(props: CustomZoomControlProps) {
         <div className="leaflet-control-zoom leaflet-bar leaflet-control">
           <a
             className={'leaflet-control-zoom-in' + disableMaxZoomButton}
-            href="#"
+            href="/#"
```

Comment:
Why was this needed?

Reply:
@bobular It is simply because eslint complained it 😓